### DISCUSSION TITLE
2006/07/24/%E3%80%8A%E8%87%B4%E5%8A%A0%E8%A5%BF%E4%BA%9A%E7%9A%84%E4%B8%80%E5%B0%81%E4%BF%A1%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#97] [Announcements]
# 2006/07/24/%E3%80%8A%E8%87%B4%E5%8A%A0%E8%A5%BF%E4%BA%9A%E7%9A%84%E4%B8%80%E5%B0%81%E4%BF%A1%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

在封建社会里，统治者们（包括君王、地主、家长、帮会头领等等）及其御用文人们，搞出了一大套理论，来要求被管理者对自己效忠。并且把对自己的忠诚提升到了道德和法律的层面，让被统治者不得不忠于自己、为自己卖命。封建制度垮台后，这些理论也遭到了摒弃。

https://blog.qizhen.xyz/2006/07/24/%E3%80%8A%E8%87%B4%E5%8A%A0%E8%A5%BF%E4%BA%9A%E7%9A%84%E4%B8%80%E5%B0%81%E4%BF%A1%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

竹贤
2006年07月27日 09:06 编辑
任何人，无论在何种岗位和职位，其工作的责任心、主动性、创造性、艺术性是更多更快地获取工作经验、提高工作能力的必要基础，继而才能获得他人（上司、同僚、下属）认可，才可能改变自己的命运。

回复
永青
2006年07月25日 09:31 编辑
双赢双赢！
